packages/core/src/state/NetworkManager.ts

```diff
@@ -101,6 +92,10 @@ export default class NetworkManager implements Manager {
     };
   }
 
+  skipLogging(action: ActionTypes) {
+    return action.type === FETCH_TYPE && action.meta.key in this.fetched;
```

Comment:
will the key ever possibly be `toString` or something else on Object.prototype?

Reply:
unlikely but possible since this is user set

Reply:
updated with hasownproperty